x-pack/auditbeat/module/system/login/login_linux_amd64_test.go

```diff
@@ -181,7 +181,7 @@ func TestBtmp(t *testing.T) {
 
 	config := getBaseConfig()
 	config["login.wtmp_file_pattern"] = ""
-	config["login.btmp_file_pattern"] = "./testdata/btmp*"
+	config["login.btmp_file_pattern"] = "./testdata/btmp_*"
```

Comment:
This was needed so that the new arm64 file would not be read.

Reply:
Minor nit, can we rename that pattern to `btmp_ubuntu` or something so it's a little more clear what it's reading? Or just  rename the test file to `btmp_amd` or something?